pages/price-feeds/contract-addresses/solana.mdx

```diff
@@ -1,7 +1,9 @@
+import CopyAddress from "../../../components/CopyAddress";
+
 # Price Feed Program Addresses on Solana
 
-| Network        | Program address                                |
-| -------------- | ---------------------------------------------- |
-| Solana Mainnet | `FsJ3A3u2vn5cTVofAjvy6y5kwABJAqYWpe4975bi2epH` |
-| Solana Devnet  | `gSbePebfvPy7tRqimPoVecS2UsBvYv46ynrzWocc92s`  |
-| Solana Testnet | `8tfDNiaEyrV6Q1U4DEXrEigs9DoDtkugzFbybENEbCDz` |
+| Network        | Program address                                                                                                                                                                 |
+| -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Solana Mainnet | <CopyAddress address={`FsJ3A3u2vn5cTVofAjvy6y5kwABJAqYWpe4975bi2epH`} url="https://explorer.solana.com/address/FsJ3A3u2vn5cTVofAjvy6y5kwABJAqYWpe4975bi2epH" />                 |
```

Comment:
url is optional but I figured I'll add it here since the other networks have url linked to them